tests/consensus/test_soft_voided4.py

```diff
@@ -43,7 +43,7 @@ def test_soft_voided(self):
 
         gen_tx2 = self.simulator.create_tx_generator(manager2, rate=10 / 60., hashpower=1e6, ignore_no_funds=True)
         gen_tx2.start()
-        self.simulator.run(900)
+        self.simulator.run(950)
```

Comment:
I'm assuming this had to change because the case changed slightly se it needed to run for longer to produce a similar situation. Right?

Reply:
Yeah, this sentence in the description talks about this:

> test_soft_voided4 also required a increase in the simulator running time, to make it generate a tx that wasn't confirmed by a block yet, which was the only way I could make it work.

Basically the change was that there were some blocks as the last generated things in the new blockchain, and this was apparently causing the test case to break for any transaction I chose.

After I increase the simulation run a little, I managed to get some additional transactions after these blocks, which were still unconfirmed by any block. Choosing these transactions made the test case work again.

Reply:
You won't need this change after PR https://github.com/HathorNetwork/hathor-core/pull/488 .